src/user.ts

```diff
@@ -13,11 +17,33 @@ export function addUser(user: User) {
 
 export function loginUser(email: string, password: string) {
   const user = getUsers().find(user => user.email == email && user.password == password);
-  if(user){
-        localStorage.setItem("current_user", JSON.stringify(user));
-        router.push("/");
-        console.log("The login was unsuccessful.")
-        return;
+  if (user) {
+    localStorage.setItem("current_user", JSON.stringify(user));
+    loadCurrentUser();
+    router.push("/");
+    console.log("The login was unsuccessful.")
```

Comment:
```suggestion
    console.log("The login was successful.")
```
In der Annahme, dass router.push nicht die Codeausführung abbricht, sollte das hier wohl eher `successful` sein, nicht?